L30-cpp-server-load-reporting.md

```diff
@@ -0,0 +1,42 @@
+﻿C++ API changes for Server Load Reporting
+----
+* Author(s): AspirinSJL
+* Approver: markdroth, vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2018-07-13
+* Discussion at: TBA
+
+## Abstract
+
+As part of the load balancing functionality, the server-side load reporting service is to be introduced. A `LoadReportingServiceServerBuilderOption` will be used to enable that service. A helper function `AddLoadReportingCost()` can be used to report the customized call metrics.
```

Comment:
What class or namespace do these new objects and functions belong to?

Reply:
Added.